sdcm/cluster_k8s/__init__.py

```diff
@@ -1063,6 +1063,16 @@ def operator_pod_status(self):
         pods = KubernetesOps.list_pods(self, namespace=SCYLLA_OPERATOR_NAMESPACE)
         return pods[0].status if pods else None
 
+    @property
+    def scylla_pod_names(self):
+        pods = KubernetesOps.list_pods(self, namespace=SCYLLA_NAMESPACE)
+        return [item.metadata.name for item in pods.items] if pods else None
+
+    @property
+    def scylla_services_names(self):
+        services = KubernetesOps.list_services(self, namespace=SCYLLA_NAMESPACE)
+        return [item.metadata.name for item in services.items] if services else None
```

Comment:
We should not get all the pods and services in namespace. We should get specific ones for our scylla cluster filtering by labels:

```
$ kubectl get svc -n scylla -l scylla/cluster=sct-cluster
$ kubectl get pods -n scylla -l scylla/cluster=sct-cluster
```

BUT, I don't think we should add it here at all. It is required in functional tests only, not in the deployment itself or nemsis.

So, if you want to have such functions, just add to one of the functional tests modules

Reply:
fixed